apps/agreement/contracts/Agreement.sol

```diff
@@ -712,6 +744,40 @@ contract Agreement is IAgreement, AragonApp {
 
     // Internal fns
 
+    /**
+    * @dev Pay transactions fees required for new actions
+    * @param _settingId Identification number of the setting being queried
+    * @param _disputable Address of the Disputable app, used to determine fees
+    * @param _submitter Address of the user that has submitted the action
+    * @param _actionId Identification number of the action to be paid for
+    */
+    function _payAppFees(uint256 _settingId, IDisputable _disputable, address _submitter, uint256 _actionId) internal {
```

Comment:
I'm wondering if we should take a storage pointer instead, since most of our internal functions now use them directly.